whotracksme/website/utils.py

```diff
@@ -1,5 +1,21 @@
+from datetime import datetime
+import json
+import pathlib
+import os
 
-def print_progress(text, default_space=40):
-    print("{} {:{}} done".format(text, "." * (default_space - len(text)), default_space - len(text)))
+class DateTimeEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, datetime):
+            return o.isoformat()
+
+        return json.JSONEncoder.default(self, o)
 
+def write_json(aPath, aDict):
+    pathlib.Path(os.path.dirname(aPath)).mkdir(parents=True, exist_ok=True)
+    jsonString = json.dumps(aDict, cls=DateTimeEncoder, indent=2)
+    jsonFile = open(aPath, "w")
```

Comment:
Stylistic:

* prefer camel case variable names
* in python, it is common to wrap it with `with`

```
with open('somefile.txt', 'a') as f:
    f.write('Hello\n')
```